packages/authentication/src/__tests__/acceptance/keys.ts

```diff
@@ -0,0 +1,25 @@
+import {BindingKey} from '@loopback/context';
```

Comment:
Nitpick: I would suggest moving `keys.ts` to a folder called `fixtures`, same for `services` and `strategies`.